File: FirstGroup/bmi.py
# Kullanıcıdan kilo ve boy bilgilerini alın
kilo = float(input("Kilonuzu kilogram cinsinden girin: "))
boy = float(input("Boyunuzu metre cinsinden girin: "))

# Beden Kitle Endeksi (BMI) hesaplama formülü
bmi = kilo / (boy ** 2)

# Sonucu ekrana yazdır
print("Beden Kitle Endeksiniz (BMI):", bmi)

# BMI değerini yorumlayarak sonucu ekrana yazdır
if bmi < 18.5:
    print("Zayıfsınız.")
elif 18.5 <= bmi < 24.9:
    print("Normal kilodasınız.")
elif 25 <= bmi < 29.9:
    print("Fazla kilolusunuz.")
else:
    print("Obezsiniz.")

File: FirstGroup/bmi.js
const bmi = (kilo, boy) => {
  let bmi = kilo / (boy * boy);
  if (bmi < 18.5) {
    return "Zayıf";
  } else if (bmi >= 18.5 && bmi < 24.9) {
    return "Normal";
  } else if (bmi >= 24.9 && bmi < 29.9) {
    return "Fazla Kilolu";
  } else {
    return "Obez";
  }
};
console.log(bmi(70, 1.6));
